### DISCUSSION TITLE
WebDAV-Push discussion room in Matrix

### DISCUSSION THREAD

**Discussion by julianfoad** [bitfireAT/webdav-push#5] [General]
Hooray for this initiative! I was just asking the [UnifiedPush folks](https://matrix.to/#/#unifiedpush:matrix.org) if anything like this exists, and Kate of [nextcloud-neon](https://github.com/provokateurin/nextcloud-neon) pointed me here, and here you are making it!

I want to encourage interested people in the relevant projects (UnifiedPush, WebDAV, JMAP, etc.) to link up and discuss WebDAV-Push. I think a matrix room is great for that. It's worked really well for the UnifiedPush project, cross-pollinating among the various groups involved in that ([NextPush](https://codeberg.org/NextPush), matrix, fediverse, etc.).

Of course anybody can create a matrix room about anything any time, but I don't want to step on anybody's toes, so how would you folks feel about this? Are you already on matrix, and if not would you be open to it with a bit of hand-holding?

*[Edits: corrected attributions re. Kate/nextcloud-neon/NextPush. Thanks to S1m.]*

**Comment by rfc2822** [comment]
Hi,

Glad to hear from you and that there's interest!

We have already created a Matrix room: _removed obsolete room ID_

Does that URL work for you? I have no Matrix experience, I just managed to create the room and link it to our Nextcloud Talk using Matterbridge ;)

**Reply by julianfoad** [reply]
Thank you! It nearly works. It's a valid matrix address, but it says "you are not invited to this room" which I think means you have set the room permission to invite-only rather than public. If you are happy to set it to public, I think that would be best.

**Reply by rfc2822** [reply]
Should be public now. Can you now join?

**Reply by rfc2822** [reply]
Please use this new room: https://matrix.to/#/#webdav-push:matrix.org

I have also added it to README.md
